receiver/filelogreceiver/README.md

```diff
@@ -112,3 +121,27 @@ receivers:
           layout: '%Y-%m-%d %H:%M:%S'
 ```
 
+## Example - Reading from a log file
```

Comment:
I would say that the value in this one is to show how to parse plain text log files instead of json files. It is showing the extra work that is necessary to parse the file using regexes.

Reply:
Ok, that's fair. Let's provide a more specific description of the use case then. "Tailing a plaintext file" perhaps?